sekoiaio_connector.py

```diff
@@ -0,0 +1,398 @@
+            "Authorization": f"Bearer {self.api_key}"
+        }
+        ret_val, response = self._make_rest_call(
+            "/v2/inthreat/indicators", action_result, params=params, headers=headers
+        )
+
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        self.save_progress("Great, we get response from the endpoint !!")
+        # Add the response into the data section
+        action_result.add_data(response.get("items", []))
+
+        # Add a dictionary that is made up of the
+        # most important values from data into the summary
+        summary = action_result.update_summary({})
+        summary["num_data"] = action_result.get_data_size()
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_get_indicator_context(self, param):
+        self.save_progress(
+            f"In get indicator context action handler for: {self.get_action_identifier()}"
+        )
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        # This 2 required parameters to requests this endpoint
+        # Take a look at
+        # cti/develop/rest_api/intelligence/Indicators/operation/get_indicator_context_resource
+        value, type_ = param.get("value", ""), param.get("type", "")
+        params, headers = {"value": value, "type": type_}, {
+            "Authorization": f"Bearer {self.api_key}"
+        }
+        ret_val, response = self._make_rest_call(
+            "/v2/inthreat/indicators/context",
+            action_result,
+            params=params,
+            headers=headers,
+        )
+
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        self.save_progress("Great, we get response from the endpoint !!")
+
+        action_result.add_data(response)
+
+        # Add a dictionary that is made up of
+        # the most important values from data into the summary
+        summary = action_result.update_summary({})
+        summary["num_data"] = action_result.get_data_size()
+
+        return action_result.set_status(phantom.APP_SUCCESS)
+
+    def _handle_get_observable(self, param):
+        self.save_progress(f"In action handler for: {self.get_action_identifier()}")
+        action_result = self.add_action_result(ActionResult(dict(param)))
+
+        value, type_, limit = (
+            param.get("value", ""),
+            param.get("type", ""),
+            param.get("limit", 20),
+        )
+        params, headers = {"value": value, "type": type_, "limit": limit}, {
+            "Authorization": f"Bearer {self.api_key}"
+        }
+        # make rest call
+        ret_val, response = self._make_rest_call(
+            "/v2/inthreat/observables", action_result, params=params, headers=headers
+        )
+
+        if phantom.is_fail(ret_val):
+            return action_result.get_status()
+
+        self.save_progress("Great, we get response from the endpoint !!")
```

Comment:
Can this be removed?

Reply:
Can't understand why I have to remove the case where the request fail ?

Reply:
Would you be able to modify the message instead to be something like: `Received response from endpoint`?

Reply:
yes of course